src/main/java/com/ge/predix/log/filter/LogFilter.java

```diff
@@ -185,16 +180,6 @@ private String setZoneId(final HttpServletRequest request) {
         return zoneId;
     }
 
-    private String setCorrelationId(final HttpServletRequest request, final HttpServletResponse response) {
-        String correlationId = request.getHeader(CORRELATION_HEADER_NAME);
-        if (StringUtils.isEmpty(correlationId)) {
-            correlationId = UUID.randomUUID().toString();
-        }
-        MDC.put(CORRELATION_HEADER_NAME, correlationId);
-        response.setHeader(CORRELATION_HEADER_NAME, correlationId);
```

Comment:
Maybe we can still keep setting the X-B3-TraceID header to maintain backwards compatibility for those folk who might now depend on the header in responses?

Reply:
@akarollil see comment above